exporter/metric/prometheus/sanitize.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: why is there a "internal/sanitize.go"? check if we can delete that.
+
+package prometheus
+
+import (
+	"strings"
+	"unicode"
+)
+
+const labelKeySizeLimit = 100
```

Comment:
I'd like all this kind of stuff to live in the API or at least somewhere higher up in the SDK.